src/pages/Introduce/Introduce.scss

```
@@ -0,0 +1,61 @@
+@import "../../style/mixin.scss";
+@import "../../style/variables";
+
+* {
+  box-sizing: border-box;
+}
+
+.introducename {
+  @include flexCenter;
+
+  .container {
+    border: 1px solid #ddd;
+    border-radius: 12px;
+    padding: 30px;
+  }
+
+  .container .logobox {
+    width: 240px;
+    height: 240px;
+    margin-bottom: 30px;
+    background-color: black;
+    background-size: cover;
+    background-repeat: no-repeat;
+    border-radius: 12px;
+    text-align: center;
+  }
+
+  .container .logobox span {
+    color: #fff;
+    @include textStyle(36px, 800);
+    line-height: 240px;
+  }
```

```suggestion
  .container .logobox span {
    @include textStyle(36px, 800);
    color: #fff;
    line-height: 240px;
  }
```

해당 부분도 위에 리뷰 드린 대로 nesting 해주시고 한가지 추가적으로 피드백 드리자면
@include 부분은 스타일의 가장 최상위에 오게 작성해주세요!

---

넵!